test-testng/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.easymock</groupId>
+    <artifactId>easymock-parent</artifactId>
+    <version>3.5-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>easymock-test-testng</artifactId>
+  <name>EasyMock Test TestNG</name>
+
+  <properties>
+    <!-- No code coverage need on this project since it's only a test project -->
+    <jacoco.skip>true</jacoco.skip>
+  </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>2.18.1</version>
+        <dependencies>
+          <dependency>
+            <groupId>org.apache.maven.surefire</groupId>
+            <artifactId>surefire-testng</artifactId>
+            <version>2.18.1</version>
+          </dependency>
+        </dependencies>
+        <configuration>
+          <!-- For an unknown reason threadCount is 0 by default and fails with TestNG -->
+          <threadCount>1</threadCount>
```

Comment:
@juherr 
I checked the Travis log again and saw this (threadCount=0)
[INFO] parallel='none', perCoreThreadCount=true, threadCount=0, useUnlimitedThreads=false, threadCountSuites=0, threadCountClasses=0, threadCountMethods=0, parallelOptimized=true

I guess that some parent, or grand parent set a system property of threadCound to zero or Maven property.
Check the Travis build. There might be such source of problem.